tensor2tensor/utils/usr_dir.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2017 The Tensor2Tensor Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Utility to load code from an external directory supplied by user."""
+
+import os
+import sys
+import importlib
+import tensorflow as tf
+
+flags = tf.flags
```

Comment:
could you define the flag in the t2t-trainer as well as the t2t-datagen and have import_usr_dir take a usr_dir as an argument. yes, it'll be duplicated, but generally we try to keep flags in the main "binary" file.